File: LengthofLastWord.py
class Solution:
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        count=0
        for i in range(len(s)-1,-1,-1):
            if s[i]==' ':
                if count==0:
                    continue
                break
            count+=1
        return count
        
        
if __name__=='__main__':
    s = raw_input()
    solution=Solution()
    print(solution.lengthOfLastWord(s))

File: LengthofLastWord.cpp
#include<iostream>
#include<string.h>

using namespace std;

class Solution {
public:
    int lengthOfLastWord(string s) {
        
        int count=0;
        for(int i=s.length()-1;i>=0;i--){
            if(s[i]==' '){
                if(count==0){
                    continue;
                }
                break;
            }
            count++;
        }
        
        return count;
        
    }
};

int main(){
    string s;
    getline(cin,s);
    
    Solution *solution=new Solution();
    
    cout<<s<<endl;
    
    cout<<solution->lengthOfLastWord(s);
    
    return 0;
}
